modules/understanding-agent-install.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent/installing-with-agent.adoc
+
+:_content-type: CONCEPT
+[id="understanding-agent-install_{context}"]
+= Understanding Agent installation
+As a {product-title} user, you can leverage all the advantages of the Assisted Installer hosted service in disconnected environments. You can generate an ISO that contains all the technology required to run both a user friendly guided deployment and a fully automated one.
```

Comment:
Are we delivering the "user friendly guided deployment" in 4.12?

Reply:
Not in 4.12.

Reply:
This is outdated content and has been removed.  The preview in the jira description reflects the new commits/content.